README.md

```diff
@@ -0,0 +1,9 @@
+This repository https://github.com/conda-forge/feedstocks/ does not get updated by hand,
+rather it is an automated summary of all the individual package feedstocks which have
+their own separate repositories.
+
+If you want to file an issue with an individual package, say ``unzip``, go to the
+https://github.com/conda-forge/unzip-feedstock/issues page.
+
+If you want to file a wider issue, like requesting a new package be added, go to the
+https://github.com/conda-forge/staged-recipes/issues page.

```

Comment:
Wider issues should be filed at https://github.com/conda-forge/conda-forge.github.io, not staged-recipes, though you are right about package requests

Reply:
I can change that - or is there a good general URL we should give instead?

Reply:
Well, it is the issues page on that repo if you want to use that